docs/source/_json/types.resp

```diff
@@ -18,6 +18,11 @@ Content-Type: application/json
     "title": "DXTestDocument"
   }, 
   {
+    "@id": "http://localhost:55001/plone/@types/DXTestFolder", 
```

Comment:
@tisto I don't need DXTestFolder at all. Standard Folder and Document are enough.
I'll remove it and cleanup a bit.